packages/convex-helpers/README.md

```diff
@@ -66,7 +68,11 @@ const posts = await asyncMap(
     const comments = await getManyFrom(db, "comments", "postId", post._id);
     // many-to-many via join table
     const categories = await getManyViaOrThrow(
-      db, "postCategories", "categoryId", "postId", post._id
+      db,
+      "postCategories",
+      "categoryId",
+      "postId",
+      post._id
```

Comment:
I'm guessing you have some VSCode setting that formats markdown with relevant formatters? 

I slightly prefer the one-liner aesthetic for a readme, but this is fine

Reply:
Yeah we can revert the formatting change, it was automatic